pkg/musl

```diff
@@ -46,3 +46,11 @@ sed -i "s,PPPP,$butch_prefix," "$musl_gcc"
 #this should speed up dynamic linker path search
 mkdir -p "$dest"/etc
 printf "%s\n" "$butch_prefix"/lib > "$dest"/etc/ld-musl-$A.path
+
+## fix ssp on i?86
+case "$A" in
+	i?86)
+		LDFLAGS="$optldflags" CFLAGS="$optcflags" gcc -c "$K/__stack_chk_fail_local.c" -o __stack_chk_fail_local.o
```

Comment:
use `$CC` instead of gcc here

Reply:
oh, i just noticed that passing LDFLAGS/CFLAGS like that has no effect: these need to be expanded on the CC commandline e.g. `$CC $optcflags ... -c ...`